.github/scripts/update-sources.sh

```diff
@@ -33,11 +33,23 @@ function update_repository() {
 echo "::group::Update source code repositories"
     mkdir -p "$SOURCE_PATH"
 
+    if [[ "$PLATFORM" =~ cygwin ]]; then
```

Comment:
So far we should not have the Cygwin build workaround changes in `woarm64` branches. Unfortunatelly, building `x86_64-pc-cygwin` target from `*-cygwin` branches specified this way breaks cahcing of `aarch64-w64-mingw32` toolchain build when `woarm64` branches do not change while the  `*-cygwin` does - it leads to cache miss.

Reply:
As discussed the changes in `woarm64-cygwin` branches will be reviewed and merged to `woarm64`.